develop/CICD/Tutorials/CICD_Tutorial_Connector.md

```diff
@@ -0,0 +1,300 @@
+   > - Deployment with a local artifact requires DataMiner 10.3.0/10.3.2 or higher.
+
+   ```yml
+   CD:
+
+   runs-on: windows-latest
+   needs: CI
+   steps:
+   - uses: actions/checkout@v4
+  
+   - name: Install Package Creation
+     run: dotnet tool install -g Skyline.DataMiner.CICD.Tools.Packager
+   - name: Install DataMiner Deploy
+     run: dotnet tool install -g Skyline.DataMiner.CICD.Tools.DataMinerDeploy
+   - name: Create Protocol Package
+     run: dataminer-package-create dmprotocol "${{ github.workspace }}" --name  "protocol" --output "${{ github.workspace }}\\_PackageResults"
+   - name: Direct Agent Deployment
+     run: dataminer-package-deploy from-artifact --path-to-artifact "${{ github.workspace }}\\_PackageResults\\protocol.dmprotocol" --dm-server-location "${{ secrets.SERVER_LOCATION }}" --dm-user "${{ secrets.DATAMINER_USER }}" --dm-password "${{ secrets.DATAMINER_PASSWORD }}"
+   ```
+
+1. Commit your changes.
+
+1. Go to the *Actions* tab, and check the run of your enhanced pipeline.
+
+   The following will be performed:
+
+   1. Compilation
+
+   1. Unit testing
+
+   1. DataMiner connector validation
+
+   1. Package creation
+
+   1. Direct Agent deployment
+
+## Step 6: Add your GitHub secrets
+
+1. Go to the *Settings* tab.
+
+1. On the left, click *Secrets and variables*, and then click *Actions*.
+
+1. Add the following secrets (by clicking *New repository secret*, entering the name of the secret, and clicking *Add secret*):
+
+   1. SERVER_LOCATION
+
+   1. DATAMINER_USER
+
+   1. DATAMINER_PASSWORD
+
+## Step 7: Enjoy the results
+
+1. Go to the *Actions* tab.
+
+1. On the left, select the *.NET* workflow.
+
+1. Click *Run workflow*.
+
+You should now see your CI and CD jobs complete successfully.
+
+> [!NOTE]
+> To be granted DevOps points, take a screenshot of this successful run and either send it to <thunder@skyline.be> or upload it via the [Dojo tutorials page](https://community.dataminer.services/learning-courses-tutorials/).
+
+## What to do next
+
+### Advanced options: CI
+
+The *Skyline Communications* organization uses more than these actions. For enhanced CI, you can take a look at our [reusable workflow](https://github.com/SkylineCommunications/_ReusableWorkflows/blob/main/.github/workflows/Connector%20Master%20SDK%20Workflow.yml):
+
+1. SonarCloud static analysis to our CI.
+
+1. Ability to use our GitHub Organization private NuGet store (when added as external collaborator to the package source repository).
+
+1. Tagging is considered a release cycle. This overrides the dmprotocol version with the tag.
+
+1. A regular commit and push is considered a build cycle. This adds a *_Bx* suffix to the dmprotocol version (*x* being the run number of the pipeline).
+
+1. The dmprotocol is uploaded as a downloadable artifact in GitHub.
```

Comment:
@janstaelensskyline 
These list items seem to be mix of information and things that happen in the workflow. Given the numbered list format, I'd expect each list item to be a step in the workflow. If this should indeed be the case, could you rephrase the steps so that each item indeed describes an action that happens in the workflow? Or if that is not what you meant, could you use a bulleted list instead and rephrase the lines without verb (i.e. the first 2) so that they become full sentences like the rest of the list?